### PR TITLE
Bugfix/add new at nil on scope change

### LINKED ISSUE #211: add_new_at nil with scope causes NoMethodError
```
acts_as_list scope: :brand_category, add_new_at: nil
```

When scope was set, error raised:

```
E, [2016-07-08T13:59:17.135900 #7403] ERROR -- : NoMethodError: undefined method `add_to_list_' for #<Item:0x007fea3942c748>
Did you mean?  add_to_transaction
  /vendor/bundle/ruby/2.3.0/gems/activemodel-4.2.6/lib/active_model/attribute_methods.rb:433:in `method_missing'
  /vendor/bundle/ruby/2.3.0/gems/acts_as_list-0.7.4/lib/acts_as_list/active_record/acts/list.rb:476:in `check_scope'
```

https://github.com/swanandp/acts_as_list/blob/master/lib/acts_as_list/active_record/acts/list.rb#L474

### PR DESCRIPTION
**Pull request by greatghoul**
To fix #211 211

### CODE REVIEW

File: lib/acts_as_list/active_record/acts/list.rb [hunk 1]
```diff
@@ -471,7 +471,9 @@ def check_scope
               send('decrement_positions_on_lower_items') if lower_item
               cached_changes.each { |attribute, values| self[attribute] = values[1] }
 
-              send("add_to_list_#{add_new_at}")
+              if add_new_at.present?
+                send("add_to_list_#{add_new_at}")
```
Comment (brendon) [inline lib/acts_as_list/active_record/acts/list.rb]: Can you make this a one-liner?

`send("add_to_list_#{add_new_at}") if add_new_at.present?`

**Comment by brendon** [comment]
Good catch on this one :) Thanks for the test and fix. Looks good to me. If you can just make that one change I'll merge it for you.

**Comment by greatghoul** [comment]
@brendon I fixed the coding style.

**Comment by brendon** [comment]
Thanks @greatghoul :) All merged.

**Comment by afn** [comment]
Thanks for fixing this!

Can we get a new version pushed to rubygems as well?

**Comment by brendon** [comment]
Yep that's not a problem. I'll push a new version now.

**Pull request merged**